test/test-PerfMeter.js

```diff
@@ -0,0 +1,162 @@
+/* globals Services */
+
+"use strict";
+const simplePrefs = require("sdk/simple-prefs");
+
+const {before, after} = require("sdk/test/utils");
+const tabs = require("sdk/tabs");
+const {setTimeout} = require("sdk/timers");
+const {ActivityStreams} = require("lib/ActivityStreams");
+const {Cu} = require("chrome");
+Cu.import("resource://gre/modules/Services.jsm");
+
+let ACTIVITY_STREAMS_URL;
```

Comment:
I frankly don't see it as a major issue, given the test context. 
We have used the exact same trick in TabTracker  https://github.com/mozilla/activity-streams/blob/master/test/test-TabTracker.js#L15, and I think it's acceptable as it saves a bit of time not to pass the exact same url around.   It does make the test code more readable to me, but I will be happy to change if you disagree.